actors/miner/src/lib.rs

```diff
@@ -1635,6 +1656,8 @@ impl Actor {
             let sector_weight_for_deposit = qa_power_max(info.sector_size);
             let deposit_req = pre_commit_deposit_for_power(&reward_stats.this_epoch_reward_smoothed, &power_total.quality_adj_power_smoothed, &sector_weight_for_deposit);
 
+            let mut precommited_sectors = vec![];
```

Comment:
We already have a sector_numbers variable. It's a bitfield, so slightly inefficient, but still easily iterable. So this extra collection is unnecessary.

Reply:
Done.